src/main/java/Sakura/Event.java

```diff
@@ -0,0 +1,46 @@
+package Sakura;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event as a task in the list.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime at;

```

Comment:
Hi Shawn, it was not immediately obvious what `at` meant. I would like to suggest using a more descriptive name, such as `startTime`.
```suggestion
    protected LocalDateTime startTime;
```